diesel/src/macros/mod.rs

```diff
@@ -833,8 +845,12 @@ mod tests {
         use macros::tests::my_types::*;
 
         /// Table documentation
-        table_with_custom_type_and_id (a){
+        ///
+        /// some in detail documentation
```

Comment:
Good test case, but this can only check if the macro parses doc comment, not if it gets output again, right? Should be good for now, though.